haiku-apps/projectconceptor/projectconceptor-1.0.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A Haiku diagramming application like Microsoft Visio"
+DESCRIPTION="ProjectConceptor is a modular application for Haiku and BeOS \
+designed to view, edit and process information which can be represented by a \
+graph structure."
+APP="ProjectConceptor"
+HOMEPAGE="https://github.com/Paradoxianer/$APP"
+COPYRIGHT="2017 Paradoxianer, 2000-2002 Lee Thomason, 1991-2000 Be Incorporated"
```

Comment:
Sorry, but that's not how we handle [`COPYRIGHT`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#copyright). We generally have [one copyright per line]((https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#copyright)).

```diff
-COPYRIGHT="2017 Paradoxianer, 2000-2002 Lee Thomason, 1991-2000 Be Incorporated"
+COPYRIGHT="2017 Paradoxianer
+	2000-2002 Lee Thomason
+	1991-2000 Be Incorporated"

Reply:
And start the first item on its own line, according to the given [examples](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#examples). :)

Reply:
Some [metrics][0] from @fbrosson for you ;)

We should consider fixing those [examples][1] as well.

[0]: https://github.com/haikuports/haikuports/pull/1896#pullrequestreview-83766512
[1]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#examples

Reply:
If some 1,600+ cases have multiple Copyright holders on the same line and only 500 or so use the multiple-line style, shouldn't we fix the guideline document instead? After all, it's just a style and not set in stone. If the devs/admins feel strongly about one way or the other, it can be enforced by `haikuporter`, like what's done to `LICENSE`.